src/webots/nodes/utils/WbNodeOperations.cpp

```diff
@@ -170,6 +171,14 @@ WbNodeOperations::OperationResult WbNodeOperations::importNode(WbNode *parentNod
   bool isNodeRegenerated = false;
   int nodeIndex = itemIndex;
   foreach (WbNode *node, nodes) {
+    if (node->isProtoInstance()) {
+      // ensure the node was declared as EXTERNPROTO prior to import it
+      if (!WbProtoManager::instance()->isDeclaredExternProto(node->modelName())) {
+        WbLog::error(tr("In order to import the PROTO '%1', first it must be declared in the Ephemeral EXTERNPROTO list.")
```

Comment:
```suggestion
        WbLog::error(tr("In order to import the PROTO '%1', first it must be declared in the EXTERNPROTO list of the world file.")
```

Reply:
Since this message is displayed when we attempt to import a non-declared node (using a supervisor), referring to "Ephemeral EXTERNPROTO" might be more clear for the user what it needs to do (since there's a button named just that on the interface)

Reply:
Yes, I agree.